garrysmod/gamemodes/terrortown/gamemode/vgui/sb_row.lua

```diff
@@ -54,7 +54,7 @@ function PANEL:Init()
    self:SetCursor( "hand" )
 end
 
-function PANEL:AddColumn( label, func, width )
+function PANEL:AddColumn( label, func, width, _, _ )

```

Comment:
To correspond with the two new parameters in the AddColumn function of sb_main

Unless the version of lua GMod uses is really screwed up the change won't make a lick of difference.